packages/ui/src/components/Badge/Badge.test.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+
+import { Badge, largeStyleProps, smallStyleProps } from './Badge';
+
+describe('Badge', () => {
+  it('renders correctly in large size', () => {
+    const text = 'Badge Text';
+
+    render(
+      <Badge data-testid="badge" size="large">
+        {text}
+      </Badge>,
+    );
+
+    const badge = screen.getByTestId('badge');
```

Comment:
I think that `screen.getByText` will be a more suitable method to catch the Badge component. There is probably no need for the `data-testid` attribute here.